distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -52,8 +52,12 @@ org.codice.ddf.system.threadPoolSize=128
 
 # HTTPS Specific settings.  If making a Secure Connection not leveraging the HTTPS Jaav libraries and 
 # classes (e.g. if you are using secure sockets directly) then you will have to set this directly
-https.cipherSuites=TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_DSS_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA
+https.cipherSuites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_128_CBC_SHA256,TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_DSS_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA
```

Comment:
A bigger question is whether or not we should get rid of the TLS_RSA_WITH_AES_128_CBC_SHA suite since this does not provide PFS. All modern web browsers support PFS at this point and the suite can always be added back in for compatibility reasons at install time. I think my vote is to remove it.